src/app/products/product/product.component.html

```diff
@@ -0,0 +1,22 @@
+<div>
+  <span class="product__name">{{product.name}}</span>
+  <span class="product__description">{{product.description}}</span>
+  <span class="product__category">{{product.category}}</span>
+  <span class="product__ingredients">
+    <span *ngFor="let ingredient of product.ingredients; let isLast=last">
+      {{ingredient}}{{isLast ? '' : ', '}}
+    </span>
+  </span>
+  <span class="product__equivalents">
+    <span *ngFor="let equivalent of product.equivalents; let isLast=last">
+      {{getAlternative(equivalent).name}}{{isLast ? '' : ', '}}
+    </span>
+  </span>
+  <span class="product__availability">{{product.isAvailable ? '&#x2714;' : '&#x2716;'}}</span>
+  <span class="product__price">${{product.price}}</span>
+  <button
+    (click)="buyProduct()"
+    [disabled]="!product.isAvailable"
+    [ngClass]="{'product__button_available': product.isAvailable, 'product__button_unavailable': !product.isAvailable}"
```

Comment:
Is it possible to use ternary here?

Reply:
Nope, that's not how `ngClass` works:
https://angular.io/api/common/NgClass

Reply:
Got it, thanks